packages/sites/src/drafts/save-published-status.ts

```diff
@@ -10,11 +10,11 @@ import { hasUnpublishedChanges } from "./has-unpublished-changes";
  * leaving everything else on the model alone.
  *
  * @param siteOrPageModel
- * @param hubRequestOptions
+ * @param requestOptions
  */
 export function savePublishedStatus(
   siteOrPageModel: IModel,
-  hubRequestOptions: IHubRequestOptions
+  requestOptions: IHubUserRequestOptions
```

Comment:
can we do this in `createSiteModelFromTemplate`?

Reply:
note to self, this also probably needs the `IHubUserRequestOptions | IHubRequestOptions` hack